### PR TITLE
Get Matches Logged-Out User Bug

### PR DESCRIPTION
**Pull request by samuelsandoval1**
## Proposed Changes
Implemented BugFix for GetMatches Logged-Out users. The fix: When logged-out user hits get matches button, they go to the page. Get this alert: This feature works for signed-in users. Click here to learn more. User clicks the button. Redirects to index page but goes to the learn more div which is where the description of the platform exists. 

## Types of Changes
What types of changes does your code introduce to Techish Project?
_Put an x in the boxes that apply_

- [x]  Bugfix (non-breaking change which fixes an issue)
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Documentation Update (if none of the other choices apply)

## Checklist
Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.

- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  I have added necessary documentation (if appropriate)
- [ ]  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/webapp/ts/matches.js [hunk 1]
```diff
@@ -146,10 +146,6 @@ function displayNewMatchPopup(matches) {
         var jsonString = JSON.stringify(match);
         var newOption = new Option(matchString, jsonString);
         newMatchContainer.add(newOption, undefined);
-        // const optionElement = document.createElement('option');
```
Comment (jerrychangusa) [inline src/main/webapp/ts/matches.js:149]: This commented out section will confuse readers who want to modify the code. Is the commented out code needed or can it be removed? 

Please try to do a self code-review to try to catch these types of issues before submitting code for review.

Comment (samuelsandoval1) [reply src/main/webapp/ts/matches.js:149]: I don't know where that came from. The commented out code happened when I ran tsc command. All I changed was two lines in matches.ts

Comment (samuelsandoval1) [reply src/main/webapp/ts/matches.js:149]: The commented out section is being removed in that commit, so it's gone already.
It got removed bc it no longer existed in the matches.ts file. So no removal needed to be done on my end.

Comment (jerrychangusa) [reply src/main/webapp/ts/matches.js:149]: ah I see - that makes more sense. Sounds good.

File: src/main/webapp/ts/matches.ts [hunk 2]
```diff
@@ -23,8 +23,8 @@ async function loadMatches() {
             })
```
Comment (ghandre) [inline src/main/webapp/ts/matches.ts:23]: nit: L8-23 is missing indendation.

Comment (samuelsandoval1) [reply src/main/webapp/ts/matches.ts:23]: What do you mean?

Comment (samuelsandoval1) [reply src/main/webapp/ts/matches.ts:23]: Oh. formatting. I see

Comment (samuelsandoval1) [reply src/main/webapp/ts/matches.ts]: Done

Comment (samuelsandoval1) [reply src/main/webapp/ts/matches.ts]: Done

**Pull request merged**